torch/lib/c10d/ProcessGroupGloo.cpp

```diff
@@ -203,6 +205,16 @@ void setInput(O& opts, at::Tensor& tensor) {
   opts.setInput(getDataPointer<T>(tensor), tensor.numel());
 }
 
+template <typename T, typename O>
+void setInput(O& opts, at::Tensor& tensor, std::vector<size_t>& counts) {
```

Comment:
Is the only difference between this and the `setInput` on Line 214 the data type in the `counts` vector? Do we need separate ones for `size_t` and `int64_t`?

Reply:
Yes, alltoall interfaces use int64_t instead of size_t
https://github.com/pytorch/pytorch/blob/master/torch/lib/c10d/ProcessGroup.hpp#L171-L175